src/utils/config.py

```diff
@@ -1481,8 +1505,12 @@ def _prevalidate(self):
 
         super()._prevalidate()
 
+        # Validation should have already failed if required
+        if self.value in (None, ""):
+            return self.value
+
         if Moulinette.interface.type != "api":
-            if not self.value or not os.path.exists(str(self.value)):
+            if not os.path.exists(str(self.value)) or not os.path.isfile(str(self.value)):
```

Comment:
hmm what's the motivation for this change here ? Not a huge deal but why not keep the `if not self.value` ? And Why enforce `isfile` ? (`exists` should be enough ... I guess people *could* maybe want to use this kind of arg for a directory or a symlink ... ?). Though as said, not a huge deal but I'm confused :sweat_smile:

Reply:
Hum, I removed the `if not self.value` because it was raising the error even if it was optional in cli, hence the check for it before. For the enforcing I don't remember lel, i'll run the tests without to check